permits/admin.py

```diff
@@ -223,6 +228,17 @@ def get_queryset(self, request):
 
         return qs
 
+    def save_model(self, req, obj, form, change):
+        """ Set 'is_staff=True' when the saved user is in a integrator group.
+        """
+        if req.user.is_superuser:
+            obj.is_staff = False
+            for group in form.cleaned_data["groups"]:
+                if group.permitdepartment.is_integrator_admin:
+                    obj.is_staff = True
```

Comment:
If `obj.is_superadmin`, do nothing. Otherwise, superadmin could loose it's admin access

Reply:
https://docs.djangoproject.com/en/4.0/ref/contrib/auth/#django.contrib.auth.models.User.is_superuser

Reply:
Resolved by https://github.com/yverdon/geocity/pull/383/commits/7a2f14fd1dac30ea60bd329856740907910bfe15